common/i18n/resources/en.js

```diff
@@ -174,6 +174,7 @@ The FRA team fra@fao.org
     home: 'Home',
     sections: {
       overview: 'Overview',
+      messageBoard: 'Message board',
```

Comment:
we should have this translated in other languages. please check and add them in case